### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by DonnieWest** [eclipse/buildship#680]
Just looking into whether or not Android support was desired and if y'all would accept a pull request.

I noticed you all were using a technique to extend the java gradle plugin and copying the jars to the /lib directory where they're added to the Eclipse classpath file.

I propose roughly what we're doing over at https://github.com/artur-shaik/vim-javacomplete2 in the file https://github.com/artur-shaik/vim-javacomplete2/blob/master/classpath.gradle

I'm pretty sure I can add all Android related jars and exploded aars to the same lib directory as part of this build process and retain compatibility with regular ol' Java projects. Some additional things we do that may/may not be applicable here since I'm unfamiliar with how the Eclipse classpath config works:

1. After outputting the paths of the jars, instead of copying them to the lib directory we reference them in place. Exploded aars are placed in the build directory and referenced from there to prevent modifying the project in a user-visible way

2. We also gather paths to any source directories

Please let me know if any of this is desirable and I'll see what I can throw together :)

**Comment by mortalis13** [comment]
Check my comment on similar issue: https://github.com/eclipse/buildship/issues/779#issuecomment-428170018.

**Comment by bmaehr** [comment]
@mortalis13 Should the commands be added to the root or the app build.gradle?

**Comment by DonnieWest** [comment]
Ideally, this sort of thing would just be built into it like we do on the Kotlin language server or javacomplete2 - no build.gradle editting needed

**Comment by mortalis13** [comment]
@bmaehr You mean commands for the plugin?
It has a [SampleApplication](https://github.com/greensopinion/gradle-android-eclipse/tree/master/src/test/SampleApplication) with plugin `apply`s in the the `app` module.

I guess it was initially created for Android Studio projects. So it converts each module (like `app`) into Eclipse projects. Though I just used it with existing Eclipse Android projects that I needed to convert to Gradle.
